axis/settings.styl

```diff
@@ -3,59 +3,79 @@
 // ---------------
 
 // Font stacks (add your own!)
-helvetica-neue = "Helvetica Neue", HelveticaNeue, Helvetica, Arial, sans-serif
-helvetica = "Helvetica Neue", Helvetica, Arial, sans-serif
-georgia = Georgia, Cambria, "Times New Roman", Times, serif
-lucidia-grande = "Lucida Grande", Tahoma, Verdana, Arial, sans-serif
-monospace = unquote("'Bitstream Vera Sans Mono', Consolas, Courier, monospace")
-verdana = Verdana, Geneva, sans-serif
+$helvetica-neue = "Helvetica Neue", HelveticaNeue, Helvetica, Arial, sans-serif
+$helvetica = "Helvetica Neue", Helvetica, Arial, sans-serif
+$georgia = Georgia, Cambria, "Times New Roman", Times, serif
+$lucidia-grande = "Lucida Grande", Tahoma, Verdana, Arial, sans-serif
+$monospace = unquote("'Bitstream Vera Sans Mono', Consolas, Courier, monospace")
+$verdana = Verdana, Geneva, sans-serif
 
 // Default font settings
-font-stack = helvetica-neue
-font-size = 16
-font-color = #555
+$font-stack = $helvetica-neue
+$font-size = 16
+$font-color = #555
 
 // Colors (via http://clrs.cc - slightly modded)
-navy = #001F3F
-blue = #0074D9
-aqua = #7FDBFF
-teal = #39CCCC
-olive = #3D9970
-green = #2ECC40
-lime = #01FF70
-yellow = #FFDC00
-orange = #FF851B
-red = #D13F19
-maroon = #85144B
-fuchsia = #F012BE
-purple = #B10DC9
-white = #FFFFFF
-silver = #DDDDDD
-gray = #AAAAAA
-black = #222222
+$navy     = #001F3F
+$blue     = #0074D9
+$aqua     = #7FDBFF
+$teal     = #39CCCC
+$olive    = #3D9970
+$green    = #2ECC40
+$lime     = #01FF70
+$yellow   = #FFDC00
+$orange   = #FF851B
+$red      = #D13F19
+$maroon   = #85144B
+$fuchsia  = #F012BE
+$purple   = #B10DC9
+$white    = #FFFFFF
+$silver   = #DDDDDD
+$gray     = #AAAAAA
+$black    = #222222
+
+//Use as default HTML colors
+navy = $navy
+blue = $blue
+aqua = $aqua
+teal = $teal
+olive = $olive
+green = $green
+lime = $lime
+yellow = $yellow
+orange = $orange
+red = $red
+maroon = $maroon
+fuchsia = $fuchsia
+purple = $purple
+white = $white
+silver = $silver
+gray = $gray
+black = $black
 
 // Default color
-default-color = blue
+$default-color = $blue
 
 // Text highlight color
-highlight-color = blue
+$highlight-color = $blue
 
 // Custom image base path for roots mixins
-img-path = ''
+$img-path = ''
 
 // Support for old IE
-support-for-ie = false
+support-for-ie = $support-for-ie  // Needed until we port/drop nib
```

Comment:
This was slightly hacky, but no way around it with the way nib is set up currently.